### PR TITLE
Dependency optimization

### PR DESCRIPTION
**Pull request by Rider21**
~Removed `urlencode` - not used~
Migration from `qs` to `react-native-url-polyfill` (https://github.com/LNReader/lnreader/pull/1302)

### CODE REVIEW

File: src/plugins/chinese/69xinshu.ts [hunk 1]
```diff
@@ -150,7 +150,7 @@ class XinShu69 implements Plugin.PluginBase {
 
     const searchUrl = `${this.site}/modules/article/search.php`;
     const formData = new FormData();
-    // formData.append('searchkey', encode(searchTerm, 'gbk'));
+    formData.append('searchkey', encode(searchTerm, 'gbk'));
```
Comment (nyagami) [inline src/plugins/chinese/69xinshu.ts:153]: Does the search work properly in app?

Comment (Rider21) [reply src/plugins/chinese/69xinshu.ts:153]: I haven't had a chance to check it out yet

**Comment by Rider21** [comment]
I think it's not necessary, react-native already has a `URLSearchParams` function (so replacing `qs` doesn't make any difference)
The update will mostly just fix the loading of illustrations.

File: src/plugins/chinese/69xinshu.ts (hunk 1 above)
Comment (nyagami) [reply src/plugins/chinese/69xinshu.ts:153]: Its domain was changed

**Review by nyagami — APPROVED** [review]

**Pull request merged**